posts/inside-rust/2019-10-26-rustc-learning-working-group-introduction.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: "The Rustc Learning Working Group - An Introduction"
+author: Amanjeev Sethi
+description: "introduction learning working group useful links"
+team: the learning working group <https://www.rust-lang.org/governance/teams/compiler#wg-learning>
```

Comment:
```suggestion
team: the rustc learning working group <https://www.rust-lang.org/governance/teams/compiler#wg-learning>
```

(I wonder if we should rename this -- it sounds so much more general than it is)